src/VisualStudio/Core/Def/ErrorReporting/VisualStudioErrorReportingServiceFactory.cs

```diff
@@ -35,17 +34,6 @@ internal sealed class VisualStudioErrorReportingServiceFactory : IWorkspaceServi
         }
 
         public IWorkspaceService CreateService(HostWorkspaceServices workspaceServices)
-        {
-            if (_singleton == null)
-            {
-                _singleton = new VisualStudioErrorReportingService(
-                    _threadingContext,
-                    _listenerProvider,
-                    workspaceServices.GetRequiredService<IInfoBarService>(),
```

Comment:
if you no longer need the IInfoBarService, then we don't need the containing type to be a factory.  It can just directly export this workspace service directly.